Infusion121/technical-test-repo
# Tech task to add edit

## Description
This PR involves changes in add and edit modal.

## Motivation and Context
We want to add attachment details to the BE, using graphql call.
Once clicked on Add attachment, we expect to see a modal, add information and submit. Which should create a row in the the attachment list.

Similarly, once we edit the modal, and add new changes our list should get updated with the new value.

## How Has This Been Tested?

1. Login to the app
2. Go to Jobs in left panel.
3. You will get list of jobs, click edit under the Actions.
4. Go to the attachment tab, and you will see the list of attachment

Add
-  Click on New attachment
-  A modal will appear, add your detail and submit
-  List should get updated

Edit
-  Click on edit button on w.r.t row
-  A modal will appear, add new detail and submit
-  List should get updated
 
## Testing/Screenshots (if appropriate):
[screen-capture (1).webm](https://github.com/Infusion121/technical-test-repo/assets/86750977/bd84a072-c02a-42ad-ad6c-79cb926e69d3)
